contrib/completion/bash/docker

```diff
@@ -4249,6 +4254,14 @@ _docker_manifest_rm() {
 	esac
 }
 
+_docker_manifest_ls() {
```

Comment:
Please insert
```bash
	case "$prev" in
		--format)
			return
			;;
	esac
```
because `--format` has an argument and we do not want completion to trigger here.

Reply:
done